scene/main/node.cpp

```diff
@@ -1087,6 +1087,7 @@ void Node::add_child(Node *p_child, bool p_legible_unique_name) {
 
 void Node::add_sibling(Node *p_sibling, bool p_legible_unique_name) {
 	ERR_FAIL_NULL(p_sibling);
+	ERR_FAIL_NULL(get_parent());
```

Comment:
Add a descriptive error message:

```suggestion
	ERR_FAIL_NULL_MSG(get_parent(), "The target node has no parent, and therefore cannot have a sibling node added.");
```